modules/RTC/JitsiLocalTrack.js

```diff
@@ -424,11 +507,16 @@ export default class JitsiLocalTrack extends JitsiTrack {
                         logger.warn(
                             `${this}: video type has changed after unmute!`,
                             this.videoType, streamInfo.videoType);
-                        this.videoType = streamInfo.videoType;
```

Comment:
Why did you remove this line?

Reply:
By mistake, thought it was my change before adding the effects option